colibris/views/api.py

```diff
@@ -9,6 +11,14 @@ class APIView(View):
     body_schema_class = None
     query_schema_class = None
 
+    async def _iter(self):
+        await self.prepare()
+
+        return await super()._iter()
+
+    async def prepare(self):
+        self.request['body_text'] = await self.request.text()
```

Comment:
@ccrisan, it simply does not work this way. You can check the test. I'm not sure why.

Reply:
Well because you need to use it as an item as well (with square brackets) when you read it. You can't access it as an attribute.

Reply:
Ahh, ok. My bad.

Reply:
Done!